tools/SchemaManager/Program.cs

```diff
@@ -43,18 +47,35 @@ private static Parser BuildParser(ServiceProvider serviceProvider)
         return commandLineBuilder.UseDefaults().Build();
     }
 
-    private static ServiceProvider BuildServiceProvider()
+    private static ServiceProvider BuildServiceProvider(string[] args)
     {
         var services = new ServiceCollection();
 
         services.AddCliCommands();
 
-        // Add SqlServer services
         services.AddOptions();
-        services.AddHttpClient();
+
+        var builder = new ConfigurationBuilder();
+
+        builder.AddCommandLine(args);
+
+        IConfigurationRoot config = builder.Build();
+
+        services.AddOptions<CommandLineOptions>().Configure(x =>
+        {
+            x.ConnectionString = config["connection-string"];
+            x.Server = new Uri(config["server"]);
```

Comment:
This line will error out if the user doesn't pass anything to the exe. Previously when you don't pass anything, you get the following output:
```
Required command was not provided.

Usage:
  Microsoft.Health.SchemaManager [options] [command]

Options:
  --version    Display version information

Commands:
  apply        Applies the specified SQL schema update to the supplied connection string for the given server.
  available    Returns the available SQL schema versions for the given server.
  current      Returns the current SQL schema version(s) from the given server.
```

Reply:
Also, not sure if this will work with the short versions of the parameters as seen below
```
You must include a connection string.

apply:
  Applies the specified SQL schema update to the supplied connection string for the given server.

Usage:
  Microsoft.Health.SchemaManager apply [options]

Options:
  -cs, --connection-string <CONNECTION-STRING>    The connection string of the SQL server to apply the schema update.
  -s, --server <SERVER>                           The root URL of the server to connect to.
  -v, --version <VERSION>                         The versions between current version and specified version to apply.
  -n, --next <NEXT>                               The next version to apply.
  -l, --latest <LATEST>                           The versions between current version and latest available version to apply.
  -f, --force <FORCE>                             The schema migration is run without validating the schema verison.
```

Reply:
Thanks for noticing these issues. They have been fixed in the latest commit.